paper-input-container.html

```diff
@@ -453,20 +455,21 @@
 
     _onBlur: function() {
       this._setFocused(false);
+      this._handleValue(this._inputElement, true);
     },
 
     _onInput: function(event) {
-      this._handleValue(event.target);
+      this._handleValue(event.target, true);
     },
 
     _onValueChanged: function(event) {
-      this._handleValue(event.target);
+      this._handleValue(event.target, true);
     },
 
-    _handleValue: function(inputElement) {
+    _handleValue: function(inputElement, validateNow) {
```

Comment:
Ideally, I wouldn't want any other parameters to `handleValue`, so maybe it's not as important now (but would be if this method ever gets changed again)